hs_core/views/utils.py

```diff
@@ -43,3 +63,163 @@ def create_form(formclass, request):
         params = formclass(data=request.REQUEST)
 
     return params
+
+
+class HSRestAPIResponse(object):
+    def __init__(self, success=True, status=None, status_code=200, message=None, data=None):
+        self.success = success
+        self.status = status
+        self.status_code = status_code
+        self.message = message
+        self.data = data
```

Comment:
this represents the response structure we decided against, so maybe it should be removed.